src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PreferIsExactSpellingIsTrueForKnownApisTest.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.InteropServices.PreferIsExactSpellingIsTrueForKnownApisAnalyzer,
+    Microsoft.NetCore.Analyzers.InteropServices.PreferIsExactSpellingIsTrueForKnownApisFixer>;
+
+namespace Microsoft.NetCore.Analyzers.InteropServices.UnitTests
+{
+    public sealed class PreferIsExactSpellingIsTrueForKnownApisTest
+    {
+        private DiagnosticResult CA1839_DefaultRule(int location, params string[] arguments)
```

Comment:
You are only expecting one argument so it'd be better to have the test helper expecting only one argument

```suggestion
        private DiagnosticResult CA1839_DefaultRule(int location, string knownApiName)
```

Reply:
Why would that help? As far as I can tell there are only overloads accepting a object[] as a parameter for setting of arguments, so we would simply delay the array creation slightly

Reply:
Yes that's only for the sake of clarity when you are trying to add new tests for an existing analyzer, it's sometimes hard to understand what's expected.